ci/jenkinsfiles/common-lib.groovy

```diff
@@ -349,10 +349,20 @@ void uploadPackage(String version, String credential,  String connectUrl) {
   withCredentials(
     [usernameColonPassword(credentialsId: "${credential}", variable: 'CONNECT_PASS')]
   ) {
-    sh """
-      PACKAGE="nuxeo-retention-package/target/nuxeo-retention-package-${version}.zip"
-      curl -i -u "$CONNECT_PASS" -F package=@\$PACKAGE "$connectUrl"/site/marketplace/upload?batch=true
-    """
+    def status = sh(
+      returnStatus: true,
+      script:
+        """
+          PACKAGE="nuxeo-retention-package/target/nuxeo-retention-package-${version}.zip"
+          curl -o /dev/null -s -w "%{http_code}\\n" -i -u "$CONNECT_PASS" -F package=@\$PACKAGE "$connectUrl"/site/marketplace/upload?batch=true | grep '200'
```

Comment:
I think you do have to use the `grep` as the example that I gave returns only the http status, nothing else